passkey-types/src/utils/serde_workaround.rs

```diff
@@ -11,7 +11,7 @@ macro_rules! serde_workaround {
         $(#[$attr:meta])*
         pub struct $name:ident {$(
             $(#[doc=$doc:literal])*
-            #[serde(rename = $discriminant:literal$(,$default:ident)?$(,skip_serializing_if = $method:path)?$(,deserialize_with = $de:path)?)]
+            #[serde(rename = $discriminant:literal$(;$default:ident)?$(,skip_serializing_if = $method:path)?$(,deserialize_with = $de:path)?)]
```

Comment:
Would it be possible to capture something like `$discriminant:literal$(,r#$default:ident)?` instead? I don't really care either way, but if we wanted to keep the Rust syntax, that might be one way of going about it.

On another note, we've had success in libwebauthn using serde-indexed for this purpose, which may be useful to look into later.